src/Api/Issue/GithubIssueApi.php

```diff
@@ -43,26 +41,23 @@ public function open(Repository $repository, string $title, string $body, array
         }
     }
 
-    public function lastCommentWasMadeByBot(Repository $repository, $number): bool
+    public function lastCommentWasMadeByBot(Repository $repository, int $number): bool
     {
         $allComments = $this->issueCommentApi->all($repository->getVendor(), $repository->getName(), $number, ['per_page' => 100]);
         $lastComment = $allComments[count($allComments) - 1] ?? [];
 
         return $this->botUsername === ($lastComment['user']['login'] ?? null);
     }
 
-    public function show(Repository $repository, $issueNumber): array
+    /**
+     * @return array<string, mixed>
+     */
+    public function show(Repository $repository, int $issueNumber): array
     {
         return $this->issueApi->show($repository->getVendor(), $repository->getName(), $issueNumber);
     }
 
-    /**
-     * Close an issue and mark it as "not_planned".
```

Comment:
Why remove?

Reply:
Hm. Did i mess with my git history while rebasing? That would be a first 😶‍🌫️